app/stac_api/models.py

```diff
@@ -120,7 +120,7 @@ class Collection(models.Model):
     created = models.DateTimeField(auto_now_add=True)  # datetime
     updated = models.DateTimeField(auto_now=True)  # datetime
     description = models.TextField()  # string  / intentionally TextField and
-    extent_geometry = models.PolygonField(default=None, srid=4326, editable=False, null=True)
+    extent_geometry = models.PolygonField(default=None, srid=4326, editable=True, null=True)
```

Comment:
This extent_geometry should not be editable by the user and it should be `blank=True` otherwise it is added to the admin page.